src/utils/ArgsRegistry.ts

```diff
@@ -12,6 +14,8 @@ export enum sdkDebugLogsLevel {
 }
 
 /**
+ * @deprecated Pass a LogLevel to debugLogsLevel in Instabug.init instead.
```

Comment:
**suggestion:** We can use backticks to make it clear when we are referencing types or methods in doc comments, since it's harder to cross-reference these types here.

```suggestion
 * @deprecated Pass a `LogLevel` to `debugLogsLevel` in `Instabug.init` instead.
```